cluster-operator/src/test/java/io/strimzi/operator/cluster/ClusterOperatorConfigTest.java

```diff
@@ -101,4 +101,21 @@ public void testEmptyEnvVars() {
 
         ClusterOperatorConfig.fromMap(Collections.emptyMap());
     }
+
+    @Test
+    public void testAnyNamespace() {
+        Map<String, String> envVars = new HashMap<>(ClusterOperatorConfigTest.envVars);
+        envVars.put(ClusterOperatorConfig.STRIMZI_NAMESPACE, "*");
```

Comment:
Again, not harm in testing `"  *  "` is also accepted.

Reply:
No, because we do not support it anymore. It doesn't pass the regex. I'm not sure I understand why would we have special handling for different typos in the configuration. But if you think we should, I would need to adapt the validation for it.